crates/isograph_api/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "isograph_api"
```

Comment:
can we make this a bit more specific, e.g. `isograph_config`? `api` is kinda vague.

Reply:
Fixed with [`1236591` (#26)](https://github.com/isographlabs/isograph/pull/26/commits/12365914ff55d6c3839f90efabf8498be374ee1a)